ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageViewModel.kt

```diff
@@ -52,10 +62,23 @@ class PreviewImageViewModel : ViewModel() {
         updateUiState(newState)
     }
 
+    fun onLoadIntoFileSuccess(filePath: String) {
+        updateLoadIntoFileState(ImageLoadToFileSuccessState(filePath))
+    }
+
+    fun onLoadIntoFileFailed() {
+        // TODO: Do we need to display any error message to the user?
```

Comment:
This scenario should not occur as load to file triggers immediately after image is displayed on the image view and available in the cache. But it might be safe to display an error if it fails in any case. I'll discuss it with the designer.